### DISCUSSION TITLE
Error was: Unknown kind "Table". [UNSUPPORTED_KIND]

### DISCUSSION THREAD

**Discussion by DonaldEnwan** [github/codeql#13839] [Q&A] [answered]
When i try to analyse the database my code created i get this error for some .ql files. This round of scanning has taken hours to complete, i'm not sure if this is normal or not, but i have not changed any of the code that came with both the cli and repo. For some files i can make this error go away by changing the @kind tag to something else but for some after trying all the different properties i saw in the docs nothing worked and i still got this error just with a different unkown kind. How do i fix this, i feel like i shouldn't be having to change stuff that came with the downloads.

Also, this may be the reason who knows, when i try to resolve the qlpacks i get this.
codeql/yaml is found in 6 same-priority locations, so attempts to resolve it will fail:
  path\codeql-home\codeql-cli\qlpacks\codeql\javascript-examples\0.0.0\.codeql\libraries\codeql\yaml\0.1.0
  path\codeql-home\codeql-cli\qlpacks\codeql\javascript-queries\0.7.0\.codeql\libraries\codeql\yaml\0.1.0
  path\codeql-home\codeql-cli\qlpacks\codeql\python-examples\0.0.0\.codeql\libraries\codeql\yaml\0.1.0
  path\codeql-home\codeql-cli\qlpacks\codeql\python-queries\0.8.0\.codeql\libraries\codeql\yaml\0.1.0
  path\codeql-home\codeql-cli\qlpacks\codeql\yaml\0.1.0
  path\codeql-home\codeql-repo\shared\yaml

Do i need to delete all but one for each because i get this for other things as well? There are multiple javascript directories that i assume would be bad to delete if i am trying to use it for javascript ofr example. Resolve languages works fine

**Comment by smowton** [comment, marked as answer]
I think you are probably doing something like running all `.ql` files in the queries repository, but some of these have special `@kind`s that are intended for use by support tools such as IDE integration.

Instead you should run a `.qls` file found in the queries repository, such as `javascript/ql/src/codeql-suites/javascript-code-scanning.qls`, which runs the high-accuracy security queries (the default query suite for Code Scanning), or the `javascript-security-extended.qls` and `javascript-security-and-quality.qls` files in the same directory which add progressively more queries.

If this doesn't work, could you post your CLI version and the exact command you are using to try to analyse your database?

**Reply by DonaldEnwan** [reply]
Yeah it seems the main reason is that i was running all javascript queries at once as i assumed that was what you were supposed to do. The meta folder was the first place i was having issues with the table property for @kind label.
